src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -392,6 +402,43 @@ class HaskellCompletionContributor extends CompletionContributor {
     waitForResult(lookupElements)
   }
 
+  private def getIdsFromAllExportedModules(project: Project) = {
```

Comment:
You can use BrowseModuleComponent, also for external modules. Which modules to ask for, depends on the context.